### PR TITLE
modify empty tuple (genericalias repr)

### PR DESCRIPTION
**Pull request by hwi-ya**
Modify `list[]` to print `list[()]` when executing `list[()]`

### CODE REVIEW

File: Lib/test/test_genericalias.py [hunk 1]
```diff
@@ -154,13 +154,11 @@ class MyList(list):
...
         self.assertEqual(t.__parameters__, ())
 
-    # TODO: RUSTPYTHON
-    @unittest.expectedFailure
     def test_repr(self):
         class MyList(list):
             pass
         self.assertEqual(repr(list[str]), 'list[str]')
-        self.assertEqual(repr(list[()]), 'list[()]')
+        self.assertEqual(repr(list[()]), 'list[()]')
```
Comment (youknowone) [inline Lib/test/test_genericalias.py:161]: whitespace is added.

```suggestion
        self.assertEqual(repr(list[()]), 'list[()]')

```

File: vm/src/builtins/genericalias.rs [hunk 2]
```diff
@@ -98,17 +98,26 @@ impl PyGenericAlias {
...
+                "{}[{}]",
+                repr_item(self.origin.as_object().to_owned(), vm)?,
+                self.args
+                    .as_slice()
+                    .iter()
+                    .map(|o| repr_item(o.clone(), vm))
+                    .collect::<PyResult<Vec<_>>>()?
+                    .join(", ")
+            ))
+        }
```
Comment (youknowone) [inline vm/src/builtins/genericalias.rs:120]: It shares the first argument but duplicating the whole formatting code.

**Review by youknowone — CHANGES_REQUESTED** [review]

File: Lib/test/test_genericalias.py (hunk 1 above)
Comment (hwi-ya) [reply Lib/test/test_genericalias.py]: @youknowone i fixed it

File: vm/src/builtins/genericalias.rs [hunk 3]
```diff
@@ -102,12 +102,16 @@ impl PyGenericAlias {
...
             "{}[{}]",
             repr_item(self.origin.as_object().to_owned(), vm)?,
-            self.args
-                .as_slice()
-                .iter()
-                .map(|o| repr_item(o.clone(), vm))
-                .collect::<PyResult<Vec<_>>>()?
-                .join(", ")
+            if self.args.len() == 0 {
+                "()".to_string()
```
Comment (youknowone) [inline vm/src/builtins/genericalias.rs:106]:
```suggestion
                "()".to_owned()
```

**Review by youknowone — APPROVED** [review]
looks good, thank you!

**Pull request merged**